src/journal-remote/journal-upload.c

```diff
@@ -121,6 +133,108 @@ static int check_cursor_updating(Uploader *u) {
         return 0;
 }
 
+int config_parse_compression(
+                const char* unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        Compression* compression = ASSERT_PTR(data);
```

Comment:
We use `Compression *compression`, rather than `Compression* compression`. That is, space before `*`.